pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3458,7 +3532,6 @@
     },
     "System.Reflection.Emit.Lightweight": {
       "StableVersions": [
-        "4.0.0",
```

Comment:
this delete is causing the CI failure.

Reply:
Hmm, looks like these were unlisted.  I think that was due to a misdirection around how to solve ref.emit.  Folks unlisted originally, but then backed up and decided to ship a new version with not-supported assembly in 5.0.  I wonder if we should relist those...

Reply:
I see, for the time being I re added this stable version back. We can decide whether or not to re-list the package in nuget separably.